src/renderer/processor.rs

```diff
@@ -145,21 +147,19 @@ impl<'a> Processor<'a> {
         }
     }
 
-    fn render_body(&mut self, body: &'a [Node]) -> Result<String> {
-        let mut output = String::with_capacity(body.len() * 20);
-
+    fn render_body(&mut self, body: &'a [Node], w: &mut impl Write) -> Result<()> {
```

Comment:
can we rename `w` -> `writer` everwhere?

Reply:
Yes I will did it

Reply:
done